aidies-project/client/src/store/configureStore.js

```diff
@@ -11,7 +12,7 @@ const reducer = combineReducers({
     
 });
 
-const storeEnhancer = composeEnhancers(applyMiddleware(thunk));
+const storeEnhancer = composeEnhancers(applyMiddleware(thunk, logger));
```

Comment:
Reminder note for the future: remove Redux `logger` before you deploy to Heroku!